.travis.yml

```diff
@@ -23,4 +23,4 @@ deploy:
   keep-history: true
   local-dir: ./dist/styleguide
   on:
-    branch: master
```

Comment:
Sowieso enkel op master, dus branch mag blijven staan.

Reply:
Ja maar hij moet niet in gang schieten bij elke master push e, alleen als je release tag plaatst op master

Reply:
Het moet de combinatie zijn: `master` en `tag`.